docs/install.rst

```diff
@@ -81,9 +91,30 @@ package:
    This is a bit more complicated, but means you only have one installation of
    jupyter lab on your machine.
 
-In either case, to open the notebooks, navigate to the ``examples/`` directory
-under this one on your terminal and activate the environment you install jupyter
-into (``plenoptic`` for 1, ``base`` for 2), then run ``jupyter`` and open up the
-notebooks. If you followed the second method, you should be prompted to select
-your kernel the first time you open a notebook: select the one named
-"plenoptic".
+   .. attention:: This method only works with conda environments. If you are using another method to manage your python virtual environments, you'll have to use one of the other methods.
+
+3. Manually install the kernel in your virtual environment. This requires only a single jupyter install and is the most general solution (it will work with conda or any other way of managing virtual environments), but requires you to be a bit more comfortable with handling environments. Again, if you followed the instructions to create a ``conda`` environment named ``plenoptic``::
```

Comment:
I'm having trouble seeing how this is different from option 2 and what it is buying you. I'd consider explaining more.